var/spack/repos/builtin/packages/py-pil/package.py

```diff
@@ -37,7 +37,13 @@ class PyPil(Package):
 
     provides('pil')
 
-    extends('python')
+    # py-pil currently only works with Python2.
+    # If you are using Python3, add to your packages.yaml:
+    # packages:
+    #     all:
+    #         providers:
+    #             pil: [py-pillow]
+    extends('python@:2.8')
```

Comment:
Once #1869 is merged, I would change this comment to "If you are using Python 3, try using py-pillow instead."